pytest-pretty and pytest-speed are not needed to run tests

--- pydantic_core-2.30.0/pyproject.toml.orig
+++ pydantic_core-2.30.0/pyproject.toml
@@ -49,9 +49,7 @@
     'pytest',
     # pytest-examples currently depends on aiohttp via black; we don't want to build it on platforms like aarch64 musllinux in CI
     'pytest-examples; implementation_name == "cpython" and platform_machine == "x86_64"',
-    'pytest-speed',
     'pytest-mock',
-    'pytest-pretty',
     'pytest-run-parallel',
     'pytest-timeout',
     'python-dateutil',
